jina/proto/serializer.py

```diff
@@ -127,3 +127,30 @@ def FromString(x: bytes):
         sp.ParseFromString(x)
 
         return sp
+
+
+class JinaInfoProto:
+    """Since the serializer is replacing the `jina_pb2 to know how to exactly serialize messages, this is just a placeholder that
```

Comment:
```suggestion
    """Since the serializer is replacing the `jina_pb2` to know how to exactly serialize messages, this is just a placeholder that
```